predicates/traffic/segment_test.go

```diff
@@ -89,16 +89,21 @@ func TestTrafficSegmentMatch(t *testing.T) {
 }
 
 func TestTrafficSegmentMinEqualsMax(t *testing.T) {
-	pp := eskip.MustParsePredicates(`TrafficSegment(0.5, 0.5)`)
-	require.Len(t, pp, 1)
-
-	spec := traffic.NewSegment()
-	p, err := spec.Create(pp[0].Args)
-	require.NoError(t, err)
+	for _, minMax := range []float64{
+		0.0,
+		1.0 / 10.0, // can not be represented exactly as float64
```

Comment:
minor: what's the problem?
```
rzavodskikh@ZALANDO-39125:/tmp$ cat main.go 
package main

import "fmt"

func main() {
	fmt.Printf("%+v\n", 1.0 / 10.0 == 0.1)
}
rzavodskikh@ZALANDO-39125:/tmp$ go build main.go 
rzavodskikh@ZALANDO-39125:/tmp$ ./main 
true
```

Reply:
https://go.dev/play/p/Hm0ja99z5mr
```go
package main

import "fmt"

func main() {
	fmt.Printf("%0.40f", 1.0/10.0)
}
```
```
0.1000000000000000055511151231257827021182
Program exited.
```

Reply:
See also https://en.wikipedia.org/wiki/Floating-point_arithmetic#Accuracy_problems